inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -32,26 +32,25 @@
 import requests
 from backports.tempfile import TemporaryDirectory
 from flask import current_app
-from lxml.etree import XMLSyntaxError
-from wand.exceptions import DelegateError
-from werkzeug import secure_filename
-
 from inspire_dojson import marcxml2record
 from inspire_schemas.builders import LiteratureBuilder
 from inspire_schemas.utils import classify_field
+from lxml.etree import XMLSyntaxError
 from plotextractor.api import process_tarball
 from plotextractor.converter import untar
 from plotextractor.errors import InvalidTarball, NoTexFilesFound
+from wand.exceptions import DelegateError
+from werkzeug import secure_filename
 
-from inspirehep.utils.record import get_arxiv_categories, get_arxiv_id
-from inspirehep.utils.url import is_pdf_link, retrieve_uri
 from inspirehep.modules.workflows.errors import DownloadError
 from inspirehep.modules.workflows.utils import (
     convert,
     download_file_to_workflow,
     with_debug_logging,
 )
-
+from inspirehep.utils.latex import decode_latex
+from inspirehep.utils.record import get_arxiv_categories, get_arxiv_id
+from inspirehep.utils.url import is_pdf_link, retrieve_uri
```

Comment:
All these import changes are not needed. You just need to add
```
from inspirehep.utils.latex import decode_latex
```

Reply:
They were all already there, my editor just decided to move them around. :+1: